source/Octopus.Client/Exceptions/InvalidSpacesLimitationParametersException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Octopus.Client.Exceptions
+{
+    public class InvalidSpacesLimitationParametersException: Exception
+    {
+        public InvalidSpacesLimitationParametersException(): base("The new spaceIds can only be a subset of the previously defined spaceIds and includeGlobal should be the same as the previous value")
```

Comment:
`includeGlobal should be the same as the previous value` this isn't true... if it was previously `true` then you should be able to change it to `false`